sync/src/synchronizer/headers_process.rs

```diff
@@ -233,7 +232,7 @@ where
 
         if log_enabled!(target: "sync", log::Level::Debug) {
             let chain_state = self.synchronizer.shared.lock_chain_state();
-            let peer_state = self.synchronizer.peers.best_known_header(self.peer);
+            let peer_state = self.synchronizer.peers.get_best_known_header(self.peer);
```

Comment:
variable name not match the value, better use `peer_best_known_header` or something close.